File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcPresentationItem/IfcSurfaceTexture/IfcBlobTexture/IfcBlobTexture.vb
Public Class IfcBlobTexture Inherits IfcSurfaceTexture
    Public RasterFormat As IfcIdentifier
    Public RasterCode As IfcBinary

    ' === WHERE CLAUSES ===
    ' RasterCodeByteStream : BLENGTH(RasterCode) MOD 8 = 0
    ' SupportedRasterFormat : SELF.RasterFormat IN ['BMP', 'JPG', 'GIF', 'PNG']
End Class

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcPresentationItem/IfcSurfaceTexture/IfcBlobTexture/IfcBlobTexture.cs
public class IfcBlobTexture : IfcSurfaceTexture
{
    public IfcIdentifier RasterFormat { get; set; }
    public IfcBinary RasterCode { get; set; }

    // === WHERE CLAUSES ===
    // RasterCodeByteStream : BLENGTH(RasterCode) MOD 8 = 0
    // SupportedRasterFormat : SELF.RasterFormat IN ['BMP', 'JPG', 'GIF', 'PNG']
}
